packages/app-accounts/src/Editor.tsx

```diff
@@ -15,8 +15,7 @@ import Forgetting from './Forgetting';
 import translate from './translate';
 
 type Props = I18nProps & {
-  allAccounts?: Array<any>,
-  onBack: () => void
+  allAccounts?: Array<Object>
```

Comment:
Sadly, Object is not much better than `any` :( We should rather either just add the `FIXME` for the any, or do it correctly with the correct typed. (Originally I was rushing, hence the `any`, when we fix it up we should not need to come back to fix `Object`)